src/app/components/carousel/carousel.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="section team-carousel" id="carousel">
+    <div class="container">
+        <div class="tim-title">
+            <h3>Our Team</h3>
+        </div>
+        <div class="page-carousel col-md-7 ml-auto mr-auto">
+            <ngb-carousel>
+                <ng-template ngbSlide #teamImg>
+                    <div class="teamImg"> 
```

Comment:
Shall we add a `ngfor ` and store data in a `json` file, that might help in the future to add more images in the carousel if needed